src/firefly/js/ui/ExampleDialog.jsx

```diff
@@ -179,7 +178,7 @@ class FieldGroupTest extends Component {
 
     constructor(props)  {
         super(props);
-        FieldGroupUtils.initFieldGroup('DEMO_FORM',exDialogReducer);
+        dispatchInitFieldGroup('DEMO_FORM',false,null,exDialogReducer);
```

Comment:
I studied it a lot, fixed it a little and I am very frustrated with the way react-redux works.  I think I want to take react-redux out completely.  The problem is not with the field group stuff. The problem is the react-redux and some of out code indiscriminately calling setState when its state has not changed. When the field group does its mount or init dispatch (which does not affect anything else) other parts of the application is calling setState without any compare. 

Any call to setState if it all the objects are equals make this message.  If the code did a shallow equals instead of relying on react there would not be a problem.

Any work around I tried to do the with field group stuff has not worked, though I am still experimenting.